controller/controller-http/src/main/java/com/pinterest/rocksplicator/controller/resource/Clusters.java

```diff
@@ -128,6 +128,19 @@ public boolean initialize(@PathParam("clusterName") String clusterName) {
   }
 
   /**
+   * Remove a given cluster. This may include removing designated tag
+   * in DB and/or writing shard config to zookeeper.
```

Comment:
I copied and pasted from the `initialize()` 's documentation: 
`/**
   * Initializes a given cluster. This may include adding designated tag
   * in DB and/or writing shard config to zookeeper.
   *
   * @param clusterName name of the cluster
   */`